src/plugins/intel_cpu/src/nodes/executors/acl/acl_eltwise.cpp

```diff
@@ -0,0 +1,173 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "acl_eltwise.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+using namespace arm_compute;
+
+TensorShape eltwiseShapeCast(const VectorDims &dims) {
+    arm_compute::TensorShape tensorShape;
+    for (std::size_t i = 0; i < dims.size(); ++i) {
+        tensorShape.set(dims.size() - i - 1, dims[i], false);
+    }
+    if (tensorShape.num_dimensions() == 0) {
+        tensorShape.set(0, 1, false);
+        tensorShape.set_num_dimensions(1);
+    }
+    return tensorShape;
+}
+
+AclEltwiseExecutor::AclEltwiseExecutor() : EltwiseExecutor() {}
+
+bool AclEltwiseExecutor::init(const EltwiseAttrs &eltwiseAttrs, const std::vector<MemoryDescPtr> &srcDescs,
+                              const std::vector<MemoryDescPtr> &dstDescs,
+                              const std::vector<EltwisePostOp> &postOps) {
+    aclEltwiseAttrs = eltwiseAttrs;
+
+    for (const auto &desc : srcDescs) {
+        if (desc->getShape().getRank() > 4)
+            return false;
+    }
+
+    for (const auto &desc : dstDescs) {
+        if (desc->getShape().getRank() > 4)
+            return false;
+    }
+
+    auto src1Dims = srcDescs[0]->getShape().getStaticDims();
+    auto src2Dims = srcDescs[1]->getShape().getStaticDims();
+    auto dstDims = dstDescs[0]->getShape().getStaticDims();
+
+    TensorInfo src1TensorInfo = TensorInfo(eltwiseShapeCast(src1Dims), 1, DataType::F32, DataLayout::NCHW);
+    TensorInfo src2TensorInfo = TensorInfo(eltwiseShapeCast(src2Dims), 1, DataType::F32, DataLayout::NCHW);
+    TensorInfo dstTensorInfo = TensorInfo(eltwiseShapeCast(dstDims), 1, DataType::F32, DataLayout::NCHW);
+
+
+
+    src1Tensor.allocator()->init(src1TensorInfo);
+    src2Tensor.allocator()->init(src2TensorInfo);
+    dstTensor.allocator()->init(dstTensorInfo);
+    std::cout << static_cast<int>(eltwiseAttrs.algorithm) << std::endl;
+
+    switch (aclEltwiseAttrs.algorithm) {
```

Comment:
In fact CPU ELtiwse node supports more operations than covered here. See https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/cpu_types.cpp#L28-L70.
The mos tthing that concerns me is unary operations (which has one input like ReLU, Swish and so on). However you are assuming that executor always has 2 inputs (e.g. srcDescs[1]->getShape()) which may lead to segfaults.